sencha-workspace/SlateAdmin/app/controller/people/Contacts.js

```diff
@@ -376,8 +381,18 @@ Ext.define('SlateAdmin.controller.people.Contacts', {
             fieldName = context.field,
             record = context.record,
             editor = context.column.getEditor(record),
+            activeEditor = editingPlugin.getActiveEditor(),
             labelEditor, valueEditor, labelStore, templateRecord, valueField, placeholder;
 
+        // allow editing multiple contact points
+        if (activeEditor && activeEditor != editor) {
+            Ext.defer(function() {
+                activeEditor.ignoreNoChange = false;
+                activeEditor.editing = true;
+                activeEditor.completeEdit();
+            }, 50);
```

Comment:
@nbey didn't we work out a better solution to this? Where did that code we worked on at your desk go?